vertx-web/src/main/java/io/vertx/ext/web/handler/impl/CSRFHandlerImpl.java

```diff
@@ -124,17 +126,18 @@ private String generateAndStoreToken(RoutingContext ctx) {
           // it's not an option to change the same site policy
           .setSameSite(CookieSameSite.STRICT));
 
-    Session session = ctx.session();
```

Comment:
I'd recommend to rename the method from `generateAndStoreToken(...)` to just `generateToken(...)` because the storing part is now handled differently, giving the method name a more real description of the functionality.